cmd/podman/compose.go

```diff
@@ -1,25 +1,23 @@
-//go:build amd64 || arm64
+//go:build !remote || amd64 || arm64
```

Comment:
the build tags seem to be messed up, `!remote` here seems wrong as you only have the compose for local commands now which means nerver macos/windows.

Reply:
To me it looks correct:
It's either !remote OR (remote && ( amd64 || arm64)) 

So I just dropped the `remote &&` as it's redundant.

Reply:
It means the podman-remote compose will not be there on non amd64, arm64 hosts which seems wrong to me

Reply:
Fair point.  That means a much more complex formula.

Reply:
If you want I can take this over as I was already looking at fixing https://github.com/containers/podman/issues/20943

Reply:
Thanks!  It's yours